taxprofiler-manuscript.qmd

```diff
@@ -129,111 +129,117 @@ A common approach to address the differences and biases between the many differe
 
 ```{#lst-example-cmd .bash lst-cap="Example nf-core/taxprofiler command for running short-read quality control, removal of host DNA and executing the k-mer based Kraken2 and marker gene alignment MetaPhlAn3 tools."}
-$ nextflow run nf-core/taxprofiler -r 1.1.0 \
--profile singularity,<institute> \
---input <samplesheet.csv> --databases <database.csv> \
---perform_shortread_qc  --shortread_qc_minlength 20 --preprocessing_qc_tool falco \
---run_host_removal --hostremoval_reference 'host_genome.fasta' \
---run_kraken2 --kraken2_save_reads \
---run_metaphlan3 \
---run_krona \
---run_profile_standardisation \
+$ nextflow run nf-core/taxprofiler \
+    -r 1.1.0 \
+    -profile singularity,<institute> \
+    --input <samplesheet.csv> \
+    --databases <database.csv> \
+    --perform_shortread_qc \
+    --shortread_qc_minlength 20 \
+    --preprocessing_qc_tool falco \
+    --run_host_removal --hostremoval_reference 'host_genome.fasta' \
+    --run_kraken2 --kraken2_save_reads \
+    --run_metaphlan3 \
+    --run_krona \
+    --run_profile_standardisation
 ```
 
 All nf-core pipelines are strictly versioned (specified with the Nextflow `-r` flag), and to ensure reproducibility, each version of the pipeline has a fixed set of software used for each step of the pipeline.
-The fixed set of software are controlled through the use of the conda package manager and containers (e.g., Docker, Apptainer [previously known as Singularity]) from the stable Bioconda [@Gruning2018-vr] and BioContainers [@Da_Veiga_Leprevost2017-gl] repositories.
-This, coupled with the intrinsic Nextflow ability to execute on most infrastructure whether that is a local laptop (resource requirements permitting), traditional HPC, as well across common cloud providers also makes nf-core/taxprofiler a very portable pipeline that can be used across many contexts.
+The fixed set of software are controlled through the use of the conda package manager or containers (e.g., Docker, Apptainer [previously known as Singularity]) from the stable Bioconda [@Gruning2018-vr] or BioContainers [@Da_Veiga_Leprevost2017-gl] repositories.
+This, coupled with the intrinsic Nextflow ability to execute on most infrastructure whether that is a local laptop (resource requirements permitting), traditional HPC, as well across common cloud providers also makes nf-core/taxprofiler a very portable pipeline that can be used in many contexts.
 
 ### Preprocessing
 
-Preprocessing steps in nf-core/taxprofiler are aimed at removing laboratory and sequencing artefacts that may influence taxonomically profiling, either for computing resource consumption and/or false-positive or false-negative classification reasons.
+Preprocessing steps in nf-core/taxprofiler are aimed at removing laboratory and sequencing artefacts that may influence taxonomic profiling, either for computing resource consumption or binary classification reasons.
```

Comment:
```suggestion
Preprocessing steps in nf-core/taxprofiler are aimed at removing laboratory and sequencing artefacts that may influence taxonomic profiling, either for computing resource consumption or and/or false-positive or false-negative classification reasons.
```

I know what I wrote was ugly but I don't have a better way of describing this atm (I wouldn't know what you meant here by 'binary classification')